src/components/Wizard/CreateVmWizard/validations/vmSettingsTabValidation.js

```diff
@@ -51,6 +52,7 @@ const validateResolver = {
   [IMAGE_URL_KEY]: asVmSettingsValidator(validateURL),
   [PROVIDER_KEY]: validateProviderDropdown,
   [MEMORY_KEY]: asVmSettingsValidator(validateMemory),
+  [HOST_NAME_KEY]: asVmSettingsValidator(validateVmLikeEntityName),
```

Comment:
I don't think host name should have the same validation as VM name

Reply:
You're right. I'll create a separate function for it. I think for the moment all that's needed is validateDNS1123SubdomainValue, correct?

Reply:
y I guess, that should be enough

Reply:
Done!